content/zh/blog/_posts/2022-05-16-volume-populators-beta.md

```diff
@@ -19,16 +19,16 @@ Ben Swartzlander (NetApp)
 Ben Swartzlander (NetApp)
 
 <!--
-The volume populators feature is now two releases old and entering beta! The `AnyVolumeDataSouce` feature
+The volume populators feature is now two releases old and entering beta! The `AnyVolumeDataSource` feature
 gate defaults to enabled in Kubernetes v1.24, which means that users can specify any custom resource
 as the data source of a PVC.
 -->
 卷填充器功能现在已经经历两个发行版本并进入 Beta 阶段！
-在 Kubernetes v1.24 中 `AnyVolumeDataSouce` 特性门控默认被启用。
+在 Kubernetes v1.24 中 `AnyVolumeDataSource` 特性门控默认被启用。
 这意味着用户可以指定任何自定义资源作为 PVC 的数据源。
 
 <!--
-An [earlier blog article](/blog/2021/08/30-volume-populators-redesigned/) detailed how the
+An [earlier blog article](/blog/2021/08/30/volume-populators-redesigned/) detailed how the
```

Comment:
Please fix line 36 for Mandarin content too.

Reply:
Done